### DISCUSSION TITLE
Several issues with opencodelists.org

### DISCUSSION THREAD

**Discussion by SRW612** [opensafely/documentation#302] [Codelists] [answered]
I am in the process of creating a number of codelists within opencodelists.org and I have run into a few issues. Hopefully it makes sense to group these into one discussion post. 

1. The first is that if I delete a draft codelist, a link remains on the 'My codelists' page, but clicking the link just takes you to the opencodelists.org homepage. I now have several of these broken links in the 'My codelists' page, which prevents me from creating a new list of the same name. Any ideas on how to remove these?
2. Is there a way to edit the name of a codelist once it has been created? I just want to fix a typo, but I can't find a way to edit the title.

**Comment by inglesp** [comment, marked as answer]
Thanks for these.

1. Sounds like a bug, but should be easy to fix.  Can you provide an example of a codelist where this is happening?
2. Not through the site at the moment, but send me an email (peter.inglesby@thedatalab.org) with requests for changes and I'll fix that up for you.

**Reply by SRW612** [reply]
Thanks Peter
1. An example from my codelists is called "Anitbiotics", and another is called "Resipartory tract infections"
2. I have just sent you an email about the relevant typo

**Reply by inglesp** [reply]
Hi Scott,

1. We've fixed the problem and these empty codelists should no longer be showing up.
2. I've fixed the typo.

Cheers,

Peter.

**Reply by SRW612** [reply]
Brilliant, thanks so much!

Scott

**Comment by nisha0119** [comment]
Hi @inglesp I hope it's ok that I tag on an additional query relating to newly created code lists. I slightly misinterpreted an element of the creating codelists video and assumed tagging individuals in the 'sign off' section would alert them to review and approve. I've since understood that 'to be reviewed' code lists should be created as an issue in codelists discussion. 

- I've thus tried to edit my metadata to remove the individuals tagged as it may mislead others, given draft is not yet approved. However, once you've indicated additional users as approvers, it doesn't allow you to remove these without replacing with new names nor allows me to leave these blank. When you submit a codelist and it says 'This version is under review', it is not yes 'published' and therefore should still allow me to carry out these edits am I right?
- on that note, do we leave this section blank in our code lists submission and return to it at a later time to add those individuals who've signed it off?

Appreciate our co-pilot @acagreen17 may have brought this up on our behalf, however, thought sharing may help others who may have encountered similar issues.

**Reply by lemanska** [reply]
Hi all, thanks @nisha0119 for this question. @inglesp please could you clarify on how the sign off happens. I am creating a code list with a colleague. She asked me to review and sign it off. However, when I access the codelist I cannot do any edits even just to metadata so I cannot sign it off. is it because: 
1. I have no necessary permissions or 
2. The code list is set up incorrectly? or 
3. The actual process is that I tell her what to edit and she eventually enters my name when I am ready to sign it off?

If I missed it somewhere in the docs, this forum or in FAQ apologies - just let me know if this has been described elsewhere. 

Thanks!

**Reply by inglesp** [reply]
Hi @lemanska, sorry for the delay in getting back to you.  To help people discover this converation in future, please could you ask your question again as a new discussion [here](https://github.com/opensafely/documentation/discussions/new)?  Please include which codelist you're having difficulties with and what username you use on OpenCodelists.

Thanks!
